hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -786,6 +798,7 @@ onOpenNetworkAckSn env openState otherParty snapshotSignature sn =
             | verify (vkey otherParty) snapshotSignature snapshot = Map.insert otherParty snapshotSignature sigs
             | otherwise = sigs
       ifAllMembersHaveSigned snapshot sigs' $ do
+        -- TODO: verify the aggregated multisig, only the individuals, or both?
```

Comment:
this TODO is resolved by this [PR](https://github.com/input-output-hk/hydra/pull/741) right?

Reply:
Yes.